pkg/cloud/aws/actuators/machine/actuator.go

```diff
@@ -314,11 +316,10 @@ func (a *Actuator) isMachineOutdated(machineSpec *v1alpha1.AWSMachineProviderSpe
 	}
 
 	if aws.BoolValue(machineSpec.PublicIP) != instanceHasPublicIP {
-		return true
+		errs = append(errs, errors.Errorf("(public IP setting cannot be mutated from \"%v\" to \"%v\")", instanceHasPublicIP, aws.BoolValue(machineSpec.PublicIP)))
```

Comment:
\"%v\" -> %q ?

Reply:
will that work for a bool?

Reply:
that doesn't work for bool :(

Reply:
Booo :(

Reply:
please use backtick delimited strings if your string contains `"`

Reply:
```suggestion
		errs = append(errs, errors.Errorf(`(public IP setting cannot be mutated from "%v" to "%v")`, instanceHasPublicIP, aws.BoolValue(machineSpec.PublicIP)))
```